Dockerfile.ubi

```diff
@@ -0,0 +1,46 @@
+FROM registry.ci.openshift.org/openshift/release:golang-1.16 as builder
```

Comment:
This file differs from `Dockerfile` only in this line, which seems like The Right Thing™

Reply:
Sorry, I didn't mean this line...